### ISSUE TITLE
Some resource providers are not available

### ISSUE THREAD

**Issue by mganeshphani** [projectkudu/AzureResourceExplorer#194]
for example Microsoft.DBForMySQL is not visible in resource explorer.. isn't this tool supposed to be dynamic? changing the code for every new provided added doesn't make sense, logic should be generic

**Comment by davidebbo** [comment]
This tool works off swagger specs. Where is the spec for the specific RP you are asking about? See https://github.com/Azure/azure-rest-api-specs/tree/master/specification

**Comment by davidebbo** [comment]
Found it: https://github.com/Azure/azure-rest-api-specs/tree/master/specification/mysql/resource-manager/Microsoft.DBforMySQL/stable/2017-12-01

Can you try the staging site: https://resources-staging.azure.com/ and see if that works?

**Comment by mganeshphani** [comment]
@davidebbo Yes staging looks fine, when this will be available in prod?... Can I assume the staging site is upto date in regards to all the resource providers available in Azure now?

**Comment by davidebbo** [comment]
@mganeshphani No, it is a manual process. It works in staging because I just updated it this morning. While ideally it would be automatic, there is not much resource to improve this tool, so it's not likely to change.

**Comment by mganeshphani** [comment]
@davidebbo 
Thanks a lot for quick action..
unfortunate that this tool isn't promoted well enough.. it does helps me a lot to take some quick actions on azure resources

**Comment by davidebbo** [comment]
@mganeshphani yes, I do wish we had more time to improve it and promote it. Still, it's in good enough shape that it's helpful to a lot of people. We just update the swaggers as people ask for it.
